SPECS/openssh/openssh.spec

```diff
@@ -143,6 +143,19 @@ sed -i 's/#PermitRootLogin prohibit-password/PermitRootLogin no/' %{buildroot}%{
 sed -i 's/#PrintMotd yes/PrintMotd no/' %{buildroot}%{_sysconfdir}/ssh/sshd_config
 sed -i 's/#PermitUserEnvironment no/PermitUserEnvironment no/' %{buildroot}%{_sysconfdir}/ssh/sshd_config
 sed -i 's/#ClientAliveInterval 0/ClientAliveInterval 120/' %{buildroot}%{_sysconfdir}/ssh/sshd_config
+sed -i 's/#ClientAliveCountMax 0/ClientAliveCountMax 0/' %{buildroot}%{_sysconfdir}/ssh/sshd_config
+
+echo "" >> %{buildroot}%{_sysconfdir}/ssh/sshd_config
+sed -i '$a# Ensure SSH LoginGraceTime is set to one minute or less\nLoginGraceTime 60' %{buildroot}%{_sysconfdir}/ssh/sshd_config
+
+echo "" >> %{buildroot}%{_sysconfdir}/ssh/sshd_config
+sed -i '$a# Ensure SSH access is limited' %{buildroot}%{_sysconfdir}/ssh/sshd_config
+sed -i '$a#AllowUsers <userlist>' %{buildroot}%{_sysconfdir}/ssh/sshd_config
+sed -i '$aDenyUsers <userlist>' %{buildroot}%{_sysconfdir}/ssh/sshd_config
+sed -i '$a#DenyGroups <grouplist>' %{buildroot}%{_sysconfdir}/ssh/sshd_config
```

Comment:
This looks like a bit overcomplicated way to append to the file (especially that mix of single lines and `\n` in the first `sed`). Let's either used `echo` for everything like we already do in the existing steps below these new lines or append everything in one go to make it even easier to read. Sample idea:

```suggestion
cat << EOF >> %{buildroot}%{_sysconfdir}/ssh/sshd_config

# Ensure SSH LoginGraceTime is set to one minute or less
LoginGraceTime 60

# Ensure SSH access is limited
#AllowUsers <userlist>
DenyUsers <userlist>
#DenyGroups <grouplist>
EOF
```

Reply:
change as suggested. I intend to comment out the DenyUsers as well but if I did so, it would not pass the asc test. As a result. I leave this DenyUser uncomment. This may look a little confusing, but I am not sure a better way to deal with this.

Reply:
This sounds fishy. We shouldn't be creating nonsensical default setting for `openssh` just to satisfy tests in a different tool depending on it. We should either set this configuration to some meaninful value or comment it out and the tests for the ASC should modify the local configuration before the tests are run.

It is somewhat worrying btw. that the ASC tests are fine with a value of `DenyUser` not set to anything existing on the running system.

Reply:
We have another option is to move this away from Mariner source and put it into post packer script like some others change which not applied to main. This way could be safer.

Reply:
> It is somewhat worrying btw. that the ASC tests are fine with a value of DenyUser not set to anything existing on the running system.

Welcome to compliance... doesn't matter if no one reads the config file as long as it has the right value in it.

I'm also not sure what the expected value here is meant to be, I'll have a quick look but I don't have background on the ASC rules, just STIG.

Reply:
I removed this from my change

Reply:
> We have another option is to move this away from Mariner source and put it into post packer script like some others change which not applied to main. This way could be safer.

Yes, in general setting the correct values during build time of a specific image sounds like a better idea. The defaults should be meaningful and modifiable, if needed. Thanks for updating the values for this PR.